### PR TITLE
feature - create alert add to cart ui

### PR DESCRIPTION
**Pull request by at-tungnguyen**
#### What's this PR do? Please give a technical summary `(required)`
- [x] create alert add to cart

### CODE REVIEW

File: MyApp/View/Controllers/Home/Drink/DrinkViewController.swift [hunk 1]
```diff
@@ -22,8 +22,6 @@ class DrinkViewController: ViewController {
         tableView.rowHeight = UITableViewAutomaticDimension
         tableView.estimatedRowHeight = 150
     }
```
Comment (at-daiho) [inline MyApp/View/Controllers/Home/Drink/DrinkViewController.swift]: WARNING - No empty line between two open or close brackets.

File: MyApp/View/Controllers/Home/Drink/DrinkViewController.swift [hunk 2]
```diff
@@ -23,7 +23,6 @@ class DrinkViewController: ViewController {
...
```
Comment (at-daiho) [inline MyApp/View/Controllers/Home/Drink/DrinkViewController.swift]: WARNING - No empty line between two open or close brackets.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 2
Serious: 0
Warnings: 2

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 3]
```diff
@@ -69,6 +69,11 @@ class DetailViewController: ViewController {
         detailTableView.rowHeight = UITableViewAutomaticDimension
         detailTableView.estimatedRowHeight = 70
     }
+    @IBAction func addToCartButtonTouchUpInside(_ sender: UIButton) {
+        let alert = UIAlertController(title: "Add to cart", message: "Your product has been added to cart.", preferredStyle: UIAlertControllerStyle.alert)
```
Comment (at-linhvo) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: 1. define text. 
2.  UIAlertControllerStyle.alert -> no need. replace = .alert

File: MyApp/View/Controllers/Detail/DetailViewController.swift [hunk 4]
```diff
@@ -69,6 +69,11 @@ class DetailViewController: ViewController {
...
+        alert.addAction(UIAlertAction(title: "Click", style: UIAlertActionStyle.default, handler: nil))
```
Comment (at-linhvo) [inline MyApp/View/Controllers/Detail/DetailViewController.swift]: same same

File: MyApp/View/Controls/CustomCells/ItemTableViewCell/ItemTableViewCell.swift [hunk 5]
```diff
@@ -8,22 +8,34 @@
...
     override func awakeFromNib() {
         super.awakeFromNib()
         addToCartButton.layer.cornerRadius = 10
         imageFoodView.layer.cornerRadius = 10
     }
 
+    // MARK: - Action
     @IBAction func addButtonTouchUpInside(_ sender: Any) {
-        print("Go To Cart")
+        let alert = UIAlertController(title: "Add to cart", message: "Your product has been added to cart.", preferredStyle: UIAlertControllerStyle.alert)
```
Comment (at-linhvo) [inline MyApp/View/Controls/CustomCells/ItemTableViewCell/ItemTableViewCell.swift]: same same.

**Comment by at-tungnguyen** [comment]
Please check my pull request. Thank you!

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

**Pull request merged**